tools/scripts/checkLicense.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+
+def extract_license(split_license):
+    assert split_license != None
+
+    license = ""
+
+    for line in split_license:
+        license += line
+
+    return license
```

Comment:
You can simplify this into single line, but at that point you might just get rid of the function altogether.

```suggestion
    return ''.join(split_license)
```

Reply:
Fixed, thanks again for teaching me something new!